dynatrace_topology/stackstate_checks/dynatrace_topology/data/conf.yaml.example

```diff
@@ -0,0 +1,33 @@
+init_config:
+
+instances:
+    # collection interval for topology
+  - min_collection_interval: 300
+
+    # Base URL of Dynatrace instance
+    # SaaS url example - https://{your-environment-id}.live.dynatrace.com
+    # Managed url example - https://{your-domain}/e/{your-environment-id}
+    url: https://test.live.dynatrace.com
+
+    # Api Token from Dynatrace platform which has access to read the topology API
+    # Docs - https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/#generate-a-token
+    token: test_token
```

Comment:
would it be possible to encrypt the token? the agent has some mechanism for encrypting stuff like this right?